workflow-cli/src/services/opensearch-snapshot.service.ts

```diff
@@ -19,22 +19,52 @@ export default class OpenSearchSnapshotService extends AwsService {
       body: JSON.stringify({
         type: 's3',
         settings: {
-          bucket: `${settings.domainName}-snapshot-${settings.accountNumber}`,
-          region: settings.region,
-          role_arn: `arn:aws:iam::${settings.accountNumber}:role/${settings.domainName}-opensearch-snapshot`,
+          'bucket': `${settings.domainName}-snapshot-${settings.accountNumber}`,
+          'region': settings.region,
+          'role_arn': `arn:aws:iam::${settings.accountNumber}:role/${settings.domainName}-opensearch-snapshot`,
+          'server-side-encryption': true,
         },
       }),
       headers: {
         'Content-Type': 'application/json',
         'host': settings.hostname,
       },
       hostname: settings.hostname,
-      path: `/_snapshot/s3-backup`,
+      path: `/_snapshot/${settings.domainName}-snapshot-${settings.accountNumber}`,
```

Comment:
Revert. Account number should not be shared unnecessarily.

Reply:
Reverted